src/manifests_workflow/input_manifests.py

```diff
@@ -164,7 +164,12 @@ def add_to_cron(self, version: str) -> None:
         with open(jenkinsfile, "r") as f:
             data = f.read()
 
-        cron_entry = f"H 1 * * * %INPUT_MANIFEST={version}/{self.prefix}-{version}.yml;TARGET_JOB_NAME=distribution-build-{self.prefix}\n"
+        build_platform_dict = {"opensearch": "linux macos windows", "opensearch-dashboards": "linux windows"}
```

Comment:
Make this a CONSTANT?

Reply:
Hey @dblock I change this dict to a class method so I believe this would be better for maintenance or we could move it to the top of this class as a fixed variable. Let me know what do you think? Thanks.

Reply:
There's nothing dynamic about this map, so since it's a constant, the convention is to declare it as such, e.g. https://github.com/opensearch-project/opensearch-build/blob/e6c5ce66ab7d702bf347da011a83129c3e5aeb82/src/test_workflow/integ_test/integ_test_runners.py#L17

Reply:
Oh I see. Yeah that makes more sense. Updating it. Thanks.